applications/forms.py

```diff
@@ -216,7 +216,7 @@ class Meta:
             'origin': 'Where are you joining us from?',
             'description': 'Why are you excited about %s?' % settings.HACKATHON_NAME,
             'projects': 'What projects have you worked on?',
-            'resume': 'Upload your resume',
+            'resume': 'Upload your resume (%s)' % ', '.join(settings.SUPPORTED_RESUME_EXTENSIONS),
```

Comment:
What happens if `SUPPORTED_RESUME_EXTENSIONS` is not set?

Reply:
But it is, I created it. if it's empty it will show only ()